src/Tus/Server.php

```diff
@@ -32,6 +32,9 @@ class Server extends AbstractTus
     /** @const string Tus Concatenation Extension */
     const TUS_EXTENSION_CONCATENATION = 'concatenation';
 
+    /** @const string Tus Content Type Extension  */
+    const HEADER_CONTENT_TYPE = 'application/offset+octet-stream';
```

Comment:
can we move this to `AbstractTus` class and use it in [Client#L428](https://github.com/ankitpokhrel/tus-php/blob/master/src/Tus/Client.php#L428) and [Server#L464](https://github.com/ankitpokhrel/tus-php/blob/master/src/Tus/Server.php#L464)